pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/DisplaySettingsRoute.java

```diff
@@ -136,6 +136,10 @@ public Object processRequest(Request request, Response response, String userId)
                 if (DDPInstanceDao.getRole(instance.getName(), DBConstants.SHOW_GROUP_FIELDS)) {
                     displaySettings.put("showGroupFields", true);
                 }
+                if (DDPInstanceDao.getRole(instance.getName(), DBConstants.HAS_ORDER_SEQUENCING)) {
```

Comment:
Now we are calling too often to get a role!
Please 
1) add a method to get a List of roles for a given instance by name
2) change class to check if list contains the role